0_download_data/correct_images.ipynb

```diff
@@ -40,9 +40,7 @@
    "source": [
     "path_to_pipeline = \"/home/jenna/NF1_SchwannCell_data/0_download_data/convert_crop_NF1_images.cppipe\"\n",
     "path_to_output = \"/home/jenna/NF1_SchwannCell_data/0_download_data/NF1_Second_Plate_Corrected\"\n",
-    "path_to_images = \"/home/jenna/NF1_SchwannCell_data/0_download_data/NF1_Second_Plate\"\n",
-    "\n",
-    "output_folder_name = \"NF1_Second_Plate_Corrected\""
+    "path_to_images = \"/home/jenna/NF1_SchwannCell_data/0_download_data/NF1_Second_Plate\""
```

Comment:
why not use relative paths instead? This ensures reproducibility on other folks machines. (Something will likely break with absolute paths)